classes/02class/exercises/c02-network09/nimmi89/README.md

```diff
@@ -0,0 +1,48 @@
+# c02-network09
+
+## Commands Execution Output
+
+- Commands for creating security groups for below table
+
+|Security Group Name|Rule|Type|Port Range|Source|Description
+|-|-|-|-|-|-|
+|public-sg|inbound|Custom TCP Rule|22|`<your-home-ip-address>`|SSH access from home to AWS|
+|private-sg|inbound|Custom TCP Rule|22|`<public-sg-id>`|SSH access only from public instances on AWS|
+
+```
+###1st security group
+aws ec2 create-security-group \
+ --group-name public-sg \
+ --description "SSH access from home to AWS" \
+ --vpc-id vpc-081afcee350babdf5
+>The command returns:
+{
+    "GroupId": "sg-040f4db337ffa4cbc"
+}
+aws ec2 authorize-security-group-ingress \
+--group-id sg-040f4db337ffa4cbc --protocol tcp \
+--port 22 --cidr 27.33.135.17/32
+
+###2nd security group
+aws ec2 create-security-group \
+--group-name private-sg \
+--description "SSH access only from public instances on AWS" \
+--vpc-id vpc-081afcee350babdf5
+>The command returns:
+{
+    "GroupId": "sg-0b64282ceb6c83fb0"
+}
+aws ec2 authorize-security-group-ingress \
```

Comment:
Not all instances may have this SG attached, so in this case, if you need to allow access from any instance in a spceific subnet(public), you would create a rule to allow access from those subnets.